src/routers/startPageRouter.ts

```diff
@@ -0,0 +1,10 @@
+import { NextFunction, Request, Response, Router } from "express";
+
+const router: Router = Router();
+const routeViews: string = "router_views/start-page";
+
+router.get("/start-page", async (req: Request, res: Response, next: NextFunction) => {
+    res.render(`${routeViews}/start-page`, {title: "Apply to register as a Companies House authorised agent"});
```

Comment:
/${routeViews}/start-page will become ${routeViews}/start-page/start-page is that expected?